packages/api/portfolioDrafts/funding/getFundingStep.ts

```diff
@@ -1,6 +1,6 @@
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
 import { ApiSuccessResponse, SuccessStatusCode } from "../../utils/response";
-import { dynamodbDocumentClient as client } from "../../utils/dynamodb";
+import { dynamodbDocumentClient as client } from "../../utils/aws-sdk/dynamodb";
```

Comment:
I am confused what this change actually does?

Reply:
Suppose I could read the docs.  🤷